ios/edge/Info.plist

```diff
@@ -68,6 +68,13 @@
 	<dict>
 		<key>NSExceptionDomains</key>
 		<dict>
+			<key>worldtimeapi.org</key>
+			<dict>
+				<key>NSIncludesSubdomains</key>
+				<true/>
+				<key>NSExceptionAllowsInsecureHTTPLoads</key>
```

Comment:
I don't believe this will work in production apps as Apple is strict about https